server/src/main/java/org/elasticsearch/cluster/coordination/NodeToolCli.java

```diff
@@ -36,6 +37,7 @@ public NodeToolCli() {
         CommandLoggingConfigurator.configureLoggingWithoutConfig();
         subcommands.put("unsafe-bootstrap", new UnsafeBootstrapMasterCommand());
         subcommands.put("detach-cluster", new DetachClusterCommand());
+        subcommands.put("repurpose", new NodeRepurposeCommand());
     }
 
     public static void main(String[] args) throws Exception {
```

Comment:
The description that we pass to super when constructing NodeToolCli no longer looks correct.
Don't forget about `ArchiveTestCase` that tests this message

Reply:
Changed it to be more generic to cover all 3 tools:
`A CLI tool to do unsafe cluster and index manipulations on current node`

See 85cf0e65edcaefb2f80280dd936b7c24839e0eae

Reply:
This description once again makes me think that detach/unsafe bootstrap and repurpose should be two different tools. However, let's leave it as it is for now.